Kwf/Assets/Cache/File.php

```diff
@@ -0,0 +1,52 @@
+<?php
+class Kwf_Assets_Cache_File
```

Comment:
As this is basically a copy of the slow cache parts of Kwf_Assets_Cache... I would prefer support switching off the fastCache directly in Kwf_Assets_Cache and don't use this extra class.